### PR TITLE
Update

### PR DESCRIPTION
**Pull request by paulinagonzalezc**
## Vet clinic database: query and update animals table

In this part of the project, I completed the following requirements:✨

* Add a column species
* Insert more indicated data
* Make indicated transactions
* Write the required queries

Here I share the screenshots with the results of the queries:

* Add a column species of type string to your animals table. Modify your schema.sql file.
Screenshot 2023-01-18 at 16 08 20

* Insert the required data
Screenshot 2023-01-18 at 16 21 06

* Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made.
Screenshot 2023-01-18 at 17 07 44

* Then roll back the change and verify that the species columns went back to the state before the transaction.
Screenshot 2023-01-18 at 17 09 05

* Inside a transaction:
** Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
** Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
** Commit the transaction.
** Verify that change was made and persists after commit.
Screenshot 2023-01-18 at 17 13 33

* Inside a transaction delete all records in the animals table, then roll back the transaction.
Screenshot 2023-01-18 at 17 16 30

* After the rollback verify if all records in the animals table still exists. After that, you can start breathing as usual
Screenshot 2023-01-18 at 17 17 35

* Inside a transaction:
** Delete all animals born after Jan 1st, 2022.
** Create a savepoint for the transaction.
Screenshot 2023-01-18 at 17 23 56

**Update all animals' weight to be their weight multiplied by -1.
Screenshot 2023-01-18 at 17 26 17

** Rollback to the savepoint
Screenshot 2023-01-18 at 17 28 28

** Update all animals' weights that are negative to be their weight multiplied by -1.
** Commit transaction
Screenshot 2023-01-18 at 17 31 35

* How many animals are there?
Screenshot 2023-01-18 at 17 39 03

* How many animals have never tried to escape?
Screenshot 2023-01-18 at 17 40 38

* What is the average weight of animals?
Screenshot 2023-01-18 at 17 42 11

* Who escapes the most, neutered or not neutered animals?
Screenshot 2023-01-18 at 17 47 52

* What is the minimum and maximum weight of each type of animal?
Screenshot 2023-01-18 at 17 50 25

* What is the average number of escape attempts per animal type of those born 
between 1990 and 2000?
Screenshot 2023-01-18 at 17 54 36

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -7,4 +7,59 @@ SELECT date_of_birth from animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
 SELECT * from animals WHERE name != 'Gabumon';
-SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Update table
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
```
Comment (t-yanick) [inline queries.sql]: - [x] You are to select species from the animals table and not all the columns as you have done in line 16. Kindly fix this.
`SELECT species from animals;`

File: queries.sql [hunk 2]
```diff
@@ -7,4 +7,59 @@ SELECT date_of_birth from animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
+---
+ROLLBACK;
+SELECT * FROM animals;
```
Comment (t-yanick) [inline queries.sql]: - [x] The same thing for line 16 and line 19.
`SELECT species from animals;`

File: queries.sql [hunk 3]
```diff
@@ -7,4 +7,59 @@ SELECT date_of_birth from animals WHERE name = 'Agumon' OR name = 'Pikachu';
...
+---
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
```
Comment (t-yanick) [inline queries.sql]: - [x] You are supposed to run this command before `Commit`
Also you should be selecting species and not all the columns as below 
`SELECT species from animals;`

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @paulinagonzalezc , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive PR

There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (paulinagonzalezc) [reply queries.sql]: Thank you @t-yanick , I will make the change.

File: queries.sql (hunk 2 above)
Comment (paulinagonzalezc) [reply queries.sql]: Thank you @t-yanick , I will make the change.

File: queries.sql (hunk 3 above)
Comment (paulinagonzalezc) [reply queries.sql]: Thank you @t-yanick , I will make the change.

**Review by Reem-lab — APPROVED** [review]
# Hi @paulinagonzalezc, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Followed Gitflow ✅
- Great working on the functionlaity ✅
- Descriptive PR ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**